### PR TITLE
[4.4] Archive content

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #38952  .

### Summary of Changes
Archive Content view does not obey all the article options. Amongst other reasons this is because it was not using the layout 

### Testing Instructions
Create a new article in a new category
set the article to archive
Create a menu item for archived articles
Test the article options in the menu item to hide/show category, date, details etc

### Before
All the optoins work except for the hide article details

### After
All works as expected

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by HLeithner** [comment]
I rebased this to j4.3 since we try to only have bug fixes in patch releases.

**Comment by brianteeman** [comment]
This is a bug fix. I deliberately submitted this to 4.2 as a bug fix.

File: components/com_content/tmpl/archive/default_items.php [hunk 1]
```diff
@@ -50,90 +51,7 @@
...
-                                <span class="icon-eye"></span>
-                                <meta itemprop="interactionCount" content="UserPageVisits:<?php echo $item->hits; ?>">
-                                <?php echo Text::sprintf('COM_CONTENT_ARTICLE_HITS', $item->hits); ?>
-                            </div>
-                        </dd>
-                    <?php endif; ?>
-                <?php endif; ?>
-                </dl>
-            </div>
+            <?php echo LayoutHelper::render('joomla.content.info_block', array('item' => $this->item, 'params' => $params, 'position' => 'above')); ?>
```
Comment (Quy) [inline components/com_content/tmpl/archive/default_items.php]:
```suggestion
            <?php echo LayoutHelper::render('joomla.content.info_block', ['item' => $this->item, 'params' => $params, 'position' => 'above']); ?>
```

File: components/com_content/tmpl/archive/default_items.php [hunk 2]
```diff
@@ -143,86 +61,7 @@
...
-                        <div class="hits">
-                            <span class="icon-eye"></span>
-                            <meta content="UserPageVisits:<?php echo $item->hits; ?>" itemprop="interactionCount">
-                            <?php echo Text::sprintf('COM_CONTENT_ARTICLE_HITS', $item->hits); ?>
-                        </div>
-                    </dd>
-                <?php endif; ?>
-            </dl>
-        </div>
+            <?php echo LayoutHelper::render('joomla.content.info_block', array('item' => $this->item, 'params' => $params, 'position' => 'above')); ?>
```
Comment (Quy) [inline components/com_content/tmpl/archive/default_items.php]:
```suggestion
            <?php echo LayoutHelper::render('joomla.content.info_block', ['item' => $this->item, 'params' => $params, 'position' => 'above']); ?>
```

**Comment by brianteeman** [comment]
array codestyle updated

**Comment by Quy** [comment]
Installed Blog Sample Data.
Created a Archived Articles menu item.
Go to the frontend.

**Comment by brianteeman** [comment]
@quy should be fixed now

**Comment by Quy** [comment]
Before PR:
38953-before

After PR:
38953-after

**Comment by brianteeman** [comment]
good spot @Quy should be fixed now

File: components/com_content/tmpl/archive/default_items.php [hunk 3]
```diff
@@ -12,6 +12,7 @@
 
 use Joomla\CMS\HTML\HTMLHelper;
 use Joomla\CMS\Language\Text;
```
Comment (Quy) [inline components/com_content/tmpl/archive/default_items.php]:
```suggestion
```

**Comment by rachelwalraven** [comment]
I have tested this item :red_circle: unsuccessfully on a32f9901358df6e3241ab3437d364b892d0919bf<br /><br />I could not reproduce the issue. All info was before patch.

Tested in Joomla 4.1 with PHP 8.1.22<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38953">issues.joomla.org/tracker/joomla-cms/38953</a>.</sub>

**Comment by TLWebdesign** [comment]
@brianteeman i tried to test this but i'm not seeing what option you're talking about that should still be displayed even when toggled to hide. Everything hides even before patch. Am i missing something?

**Comment by formfranska** [comment]
I have tested this item :white_check_mark: successfully on a32f9901358df6e3241ab3437d364b892d0919bf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38953">issues.joomla.org/tracker/joomla-cms/38953</a>.</sub>

**Comment by formfranska** [comment]
I have tested this item :white_check_mark: successfully on a32f9901358df6e3241ab3437d364b892d0919bf<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38953">issues.joomla.org/tracker/joomla-cms/38953</a>.</sub>

**Comment by obuisard** [comment]
It is a bit confusing to understand, but what I gathered is that you need to have 
image
And check if 'Details' is showing or not

**Comment by HLeithner** [comment]
This pull request has been automatically rebased to 4.4-dev.

**Comment by fgsw** [comment]
I have tested this item :red_circle: unsuccessfully on 373bc124a110d20636e44cc9a8e60d816fc73ed6<br /><br />Explanation follow soon.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/38953">issues.joomla.org/tracker/joomla-cms/38953</a>.</sub>

**Comment by Hackwar** [comment]
@brianteeman can you have a look at this PR and see if you can get this ready?

**Comment by brianteeman** [comment]
Closing for now as I dont have the available time to look into it

**Pull request closed without merging**